### PR TITLE
feat: add avalanche tokens

### PR DESCRIPTION
**Pull request by marktoda**
This commit adds avalanche tokens to the tokenlist

### CODE REVIEW

File: src/processTokens.ts [hunk 1]
```diff
@@ -132,8 +133,14 @@ export async function processTokens(
                 ...tokenConfig.chains[chainId],
             };
             const tokenData = {name, symbol, address, decimals, chainId};
-            if (logoURI) {
-                Object.assign(tokenData, {logoURI});
+            if (await doesLogoExist(primaryAddress, chainId)) {
+                Object.assign(tokenData, { 
+                    logoURI: `https://buttonwood-protocol.github.io/buttonwood-token-list/assets/tokens/${chainId}/${primaryAddress}.png`,
+                });
+            } else {
```
Comment (Fiddlekins) [inline src/processTokens.ts:140]: This breaks the checking that asset actually exists (and is why logoURIs have been added for a bunch of assets in the output tokenlist, despite them failing to load)

I think making line 140 an `else if(await doesLogoExist(primaryAddress){` would resolve this

Comment (Fiddlekins) [reply src/processTokens.ts:140]: I think we should aim to keep logoURIs that don't have an asset where they're pointing to out of the tokenlist

**Review by Fiddlekins — CHANGES_REQUESTED** [review]

File: src/processTokens.ts (hunk 1 above)
Comment (marktoda) [reply src/processTokens.ts:140]: cool sounds good

Comment (marktoda) [reply src/processTokens.ts]: fixed

**Review by Fiddlekins — APPROVED** [review]

**Pull request merged**